doc/man1/openssl-pkeyutl.pod.in

```diff
@@ -63,20 +65,26 @@ if this option is not specified.
 
 =item B<-rawin>
 
-This indicates that the input data is raw data, which is not hashed by any
-message digest algorithm. The user can specify a digest algorithm by using
-the B<-digest> option. This option can only be used with B<-sign> and
-B<-verify> and must be used with the Ed25519 and Ed448 algorithms.
+This indicates that signature input data is raw data, which for most signature
+algorithms (but not EdDSA) needs to be hashed by some message digest algorithm.
+This option can only be used with B<-sign> and B<-verify>
+and is implied by the Ed25519 and Ed448 algorithms.
+Except with EdDSA,
+the user can specify a digest algorithm by using the B<-digest> option.
+
+The B<-digest> option implies B<-rawin>.
```

Comment:
Should there bit a `HISTORY` or `NOTES` section that mentions the behaviour changes in this PR (mostly that `-digest` implying `-rawin` is new with OpenSSL 3.5 (or whichever is the oldest release to which this is backported).

Should there be some mention of `-pkeyopt digest:md` for RSA PKCS#1 when not `-rawin`?

Reply:
I've added to the HISTORY section:
```
Since OpenSSL 3.5,
the B<-digest> option implies B<-rawin>, and these two options are
no longer required when signing or verifying with an Ed25519 or Ed448 key.

```
What do you and to others think - can we backport all parts of this PR?

Reply:
> Should there be some mention of -pkeyopt digest:md for RSA PKCS#1 when not -rawin?

I'm not familiar with `-pkeyopt digest:md` and the like, so I may take some time and need to ask for more concrete input on it, in particular for being able to add suitable tests.
Should we actually consider handling `-pkeyopt digest:md` as part of this PR?